docker/scripts/mininet/multi_switch_mininet.py

```diff
@@ -79,9 +81,11 @@ def main():
     conf = manifest['targets'][args.target]
     params = conf['parameters'] if 'parameters' in conf else {}
 
+    os.environ.update(dict(map(lambda (k,v): (k, str(v)), params.iteritems())))
+
     def formatParams(s):
         for param in params:
-            s = s.replace('%'+param+'%', str(params[param]))
+            s = s.replace('{'+param+'}', str(params[param]))
```

Comment:
(There may be a python package that does shell-style variable expansion for you; I haven't looked, but it seems like something that might already exist.)